src/assets/stylesheets/styles.scss

```diff
@@ -100,17 +100,30 @@
 
 // 3. Load any custom SASS
 
+/* Custom utilities classes */
 // text styling
 .text-capitalize {
   text-transform: capitalize;
 }
 .text-break-all {
   word-break: break-all;
 }
+.text-ellipsis {
+  @include u-overflow('hidden');
+  @include u-text('no-wrap');
+  @include u-display(
+    'block'
+  ); // Use flex display on parent for inline elements.
```

Comment:
Non-blocking question: is this whitespace significant to the mixin? (Just asking for my edification here, I’m only curious!)

Reply:
Great question! The text-overflow property doesn't force an overflow to occur by itself. To make text overflow its container, you have to set other CSS properties `overflow` and `white-space`. 

For _my_ edification, I tried setting the white-space to default `wrap` and this happened 😬 
![image](https://github.com/user-attachments/assets/c1e1490a-6539-47ce-ac29-6e9c4893821d)

Reply:
Thanks @cannandev! And sorry, I don’t think my question was remotely clear: I was wondering if there’s something about the `u-display` mixin that has to be written like this, across multiple lines:

```@include u-display(
    'block'
  );```

(The previous two mixins are all written on one line, so I was just curious.) (And again: non-blocking question!)

Reply:
It's the prettier linter that runs on save. It sees the comment as part of the line length. 😞